### PR TITLE
Update variables table in custom error page guide

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description
- **What's changed?** Renames a column in the variables table. Clarifies a note about escaped HTML. Removes most inline line breaks.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-525353](https://oktainc.atlassian.net/browse/OKTA-525353)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 1]
```diff
@@ -60,22 +60,23 @@ Use the code editor to modify any HTML, CSS, or JavaScript on the error page. Se
...
 
-> **Note:** Variables with double curly braces (`{{`) return escaped HTML by default. Escaping allows you to show "special" characters in HTML. For example, `<p>hello</p>` displays as a paragraph element without the `<p>` tags. To show the tags themselves, escape the special characters by using `&lt;p&gt;hello&lt;/p&gt;`. `&lt;p&gt;` replaces `<` and `&lt;/p&gt;` replaces `>`.
-> Triple curly braces (`{{{`) are used for the `errorDescription` variable to return unescaped HTML.
+Variables with double curly braces (`{{`) return escaped HTML by default. Escaping allows you to show "special" characters in HTML. For example, `<p>hello</p>` displays as a paragraph element. The `<p>` tags don't render. For the `<p>` tags to render, escape or replace the `<p>` tags by using `&lt;p&gt; hello &lt;/p&gt;`. In this example, `&lt;p&gt;` escapes `<` and `&lt;/p&gt;` escapes `>`.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:63]: Suggested: 

For example, `<p>hello</p>` displays as a paragraph element and the `<p>` tags don't render.

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 2]
```diff
@@ -60,22 +60,23 @@ Use the code editor to modify any HTML, CSS, or JavaScript on the error page. Se
...
 
-| Variable | Description |
+Triple curly braces (`{{{`) are used for the `errorDescription` variable to return unescaped HTML.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:65]: It's a little unclear. Triple curlies are only for `errorDescription`? Can we add "only" to make it clear?

**Review by robertnicolo-okta — APPROVED** [review]
Hey Brent, had two comments and approved to not hold you up.

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md (hunk 2 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:65]: Thanks, Robert :)

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 3]
```diff
@@ -60,22 +60,23 @@ Use the code editor to modify any HTML, CSS, or JavaScript on the error page. Se
...
+| <span v-pre>`{{orgLogo}}`</span> | The logo image that has been configured for your application. You can change this logo by using the [Sign-in Configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-set-theme) option, but this changes the org logo in all instances where the variable is used, including your custom sign-in page. If you want to change only the logo image for your custom error pages, include the URL to the image instead of the variable. |
+| <span v-pre>`{{{errorDescription}}}`</span> | A detailed description of the error |
+| <span v-pre>`{{back}}`</span> | The text "Go to Homepage". When the user selects the button, they are returned to the sign-in page. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:74]: should also change "selects" to clicks in line 75/76
```suggestion
| <span v-pre>`{{back}}`</span> | The text "Go to Homepage". When the user clicks the button, they are returned to the sign-in page. |
```

**Review by susanharper-okta — APPROVED** [review]
one baby nit. approving to not hold you up.

**Review by albertchen-okta — APPROVED** [review]

**Pull request merged**